### PR TITLE
Update babel.md

### PR DESCRIPTION
**Pull request by abhaypawar**
Translated babel.md.
Please check and review @spielers and @visshaljagtap.

### CODE REVIEW

File: docs/docs/babel.md [hunk 1]
```diff
@@ -2,19 +2,20 @@
 title: Babel
 ---
 
-Gatsby uses the phenomenal project [Babel](https://babeljs.io/) to enable support for writing modern JavaScript — while still supporting older browsers.
 
-## How to specify which browsers to support
+जुन्या ब्राऊजरला समर्थन देत असताना- गॅटस्बी [babel](https://babeljs.io/) आधुनिक जावास्क्रिप्ट लिहिण्यास समर्थन देण्यासाठी वापरतो
```
Comment (spielers) [inline docs/docs/babel.md]: Javascript should be in English as it is

File: docs/docs/babel.md [hunk 2]
```diff
@@ -2,19 +2,20 @@
...
+## कोणत्या ब्राउझरचे समर्थन करावे हे निर्दिष्ट कसे कराल 
 
-This means that your JavaScript is automatically compiled to ensure it works on older browsers. Polyfills are also automatically added — no more shipping code which mysteriously breaks on older browsers!
+गॅटस्बी मुख्य ब्राउझरच्या शेवटच्या दोन आवृत्त्यांना डीफॉल्टनुसार समर्थन देते, IE 9+, तसेच कोणताही ब्राउझर ज्यात अद्याप 1% + ब्राउझर सामायिक आहे.
 
-If you only target newer browsers, see the [Browser Support](/docs/browser-support/) docs page for how to instruct Gatsby on which browsers you support and then Babel will start compiling for only these browsers.
+याचा अर्थ असा आहे की आपली Javascript जुन्या ब्राउझरवर कार्य करते हे सुनिश्चित करण्यासाठी स्वयंचलितपणे संकलित केली आहे. पॉलीफिल देखील स्वयंचलितपणे जोडली जातात- जुन्या ब्राउझरवर रहस्यमयपणे खंडित करणारा कोणताही शिपिंग कोड नाही!
 
-## How to use a custom .babelrc file
+आपण केवळ नवीन ब्राउझरवर लक्ष केंद्रित केले असल्यास, कोणत्या ब्राउझरवर समर्थित आहात यावर गॅटस्बीला कसे शिकवायचे यासाठी [Browser Support](/docs/browser-support/) दस्तऐवज पेज पहा आणि नंतर बाबेल केवळ या ब्राउझरसाठी कंपाईल करणे प्रारंभ करेल.
```
Comment (spielers) [inline docs/docs/babel.md]: babel should de in English its a term not a word

**Review by spielers — CHANGES_REQUESTED** [review]
Hey, @abhaypawar thanks once again !! please make suggested changes.

File: docs/docs/babel.md (hunk 1 above)
Comment (abhaypawar) [reply docs/docs/babel.md]: Corrected. I request you to check. 
Thanks again!

File: docs/docs/babel.md (hunk 2 above)
Comment (abhaypawar) [reply docs/docs/babel.md]: babel word corrected now. Please check.
Thank you @spielers

**Comment by abhaypawar** [comment]
Completed. Please check

**Review by spielers — APPROVED** [review]
@abhaypawar  Thanks Keep it up 👍

**Pull request merged**